plugins/pip/Makefile.am

```diff
@@ -0,0 +1,62 @@
+include $(top_srcdir)/Makefile.common
+
+VAPIDIR = $(top_srcdir)/vapi
+
+BUILT_SOURCES = libgala_pip_la_vala.stamp
+
+libgala_pip_la_LTLIBRARIES = libgala-pip.la
+
+libgala_pip_ladir = $(pkglibdir)/plugins
+
+libgala_pip_la_LDFLAGS = \
+	$(PLUGIN_LDFLAGS) \
+	$(GALA_CORE_LDFLAGS) \
+	$(top_builddir)/lib/libgala.la \
+	$(NULL)
+
+libgala_pip_la_CFLAGS = \
+	$(GALA_CORE_CFLAGS) \
+	-include config.h \
+	-w \
+	-I$(top_builddir)/lib \
+	$(NULL)
+
+libgala_pip_la_VALAFLAGS = \
+	$(GALA_CORE_VALAFLAGS) \
+	$(top_builddir)/lib/gala.vapi \
+	--vapidir $(VAPIDIR) \
+	$(VAPIDIR)/config.vapi \
+	$(NULL)
+
+libgala_pip_la_LIBADD = \
+	$(GALA_CORE_LIBS) \
+	$(NULL)
+
+libgala_pip_la_VALASOURCES = \
+	Main.vala \
+	MoveAction.vala \
+	PopupWindow.vala \
+	RoundedRectangleEffect.vala \
```

Comment:
The RoundedRectangleEffect is not used anywhere in the code and is not needed in the vala sources here. The file could be as well removed altogether.